.github/workflows/conformance-kind-proxy-embedded.yaml

```diff
@@ -3,16 +3,54 @@ name: Conformance Kind Envoy Embedded
 # Any change in triggers needs to be reflected in the concurrency group.
 on:
   pull_request:
-    paths-ignore:
-      - 'Documentation/**'
-      - 'test/**'
+    paths:
```

Comment:
just a note that if the workflow is set as required, and it's not triggered due to path filter, the PR will not be able to merge without super power.

Reply:
Ah ah! good point!